storybookjs/storybook
# CRA Example Cleanup - Info

Issue:

#1750 

## What I did

- cleaned up info examples and added new examples for different options
- turned on automatic import for `*.stories.js` so future stories could be separated easily
- will cleanup existing info usages and move old examples to a deprecated kind later

## How to test

Is this testable with jest or storyshots?
yes

Does this need a new example in the kitchen sink apps?
this is it

Does this need an update to the documentation?
no

If your answer is yes to any of these, please make sure to include it in your PR.
